lib/ansible/galaxy/data/default/collection/plugins/README.md.j2

```diff
@@ -1,3 +1,4 @@
+#SPDX-License-Identifier: MIT-0
```

Comment:
I think we need to remove this one too?

Reply:
yeah, went by extension, excluded .md .. but forgot .md.j2 ...